### DISCUSSION TITLE
Add FastDDS Gen version number in generated source code as a comment

### DISCUSSION THREAD

**Discussion by amdsobhy** [eProsima/Fast-DDS-Gen#232] [Ideas]
It is important to know the version of the FastDDS-Gen used to generate the source code. Currently there is no way to tell the version of FastDDS-Gen used to generate the source which causes issues trying to figure out which version each developer used.

I think this is a good and simple feature to be added to the generator so that developers could easily confirm which version was used to generate this source code and be able to append to it using the same generator

It would be nice if the version is added as following:

```
#define FASTDDSGEN_VERSION_MAJOR 2
#define FASTDDSGEN_VERSION_MINOR 5
#define FASTDDSGEN_VERSION_PATCH 0
```

Thank you

**Comment by JLBuenoLopez** [comment]
Moving this ticket into the Ideas discussion forum where enhancements and feature requests are discussed.
